### ISSUE TITLE
Empty fragment after rotate from landscape and 2 instances of the same fragment

### ISSUE THREAD

**Issue by Perdator** [hiteshbpatel/MultiPaneFragmentExampleAndroid#1]
If we start an app in a landscape mode first and then rotate to the portrait mode we can see empty screen UI. It causes this line of code:

if (savedInstanceState != null){
                return;
            }
If we comment it we can get issue with two instances of fragment. Is solution exists?

**Comment by hiteshbpatel** [comment]
Yes, You can design in xml file with layout and layout-land and you can work for both.
